src/sql/platform/query/common/queryManagement.ts

```diff
@@ -210,6 +210,7 @@ export class QueryManagementService implements IQueryManagementService {
 		});
 	}
 	public runQueryString(ownerUri: string, queryString: string): Thenable<void> {
+		this.addTelemetry(TelemetryKeys.RunQueryString, ownerUri);
```

Comment:
@abist can you take a look at this and decide a) if we need telemetry here, b) if there's any impact on our pipelines from this, and c) if we need a new string vs. just using the existing one? You're our telemetry expert so don't want to have any unexpected impact, thanks!

Reply:
This is fine as long as we don't include the query string itself along with it 👍